operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go

```diff
@@ -68,6 +68,16 @@ type PlatformLogsCollectionSpec struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
+// PlatformMetricsCollectionSpec defines the spec for the addon to collect and forward metrics
+// from fleet managed clusters.
+type PlatformMetricsCollectionSpec struct {
+	// Enabled defines a flag to enable/disable the platform metrics collection.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	Enabled bool `json:"enabled,omitempty"`
+}
```

Comment:
In this initial version since metrics will be taking a managed approach to metric collection I wonder if these values are the best ones to use since the equivalent for logs & traces currently correlates to an unmanaged approach where the user fully controls the root resources. I would be in favor of deliberating a bit how we want these different scenarios to be exposed to users. This correlates with  https://github.com/stolostron/multicluster-observability-addon/pull/88#issue-2658695381 maybe a discussion for the nexus syncs

Reply:
You mean that you expect different fields in the MCO CRD to have either managed or unmanaged deployments?
It would be very interesting to discuss those concepts in the next nexus sync. 
On the metrics side, I am not sure there is demand for unmanaged type of deployments 🤷 . The expressed need is to replicate what we have now. We need something in between with some managed fields and some that we can leave it to the user for fine tuning.
Let's discuss all that in the nexus sync 👍